### ISSUE TITLE
Add custom subtitle support.

### ISSUE THREAD

**Issue by Jakeverse** [insomniachi/Totoro#28]
Please add custom subtitle loading option in player. Many foreign anime fan have their own language subtitle. If you can implement this feature then it will be a big help to anime fan. We can enjoy anime in our own language.. I'm really enjoying this beautiful app except subtitle feature. And please continue adding new features and update Totoro. Also share this app in reddit so can more people will know about this great app. 

Thanks 😊

**Comment by insomniachi** [comment]
Do you want the app to find the subtitles for you?
Or, you'll find the subtitle file and you just need option to load the file?

**Comment by Jakeverse** [comment]
It will be best if you just add a option to load subtitle file. Also I would like to watch anime with fansub and it's not available in just one place so for now adding loading subtitle file option will be best for me and anyone who wanna watch with local fansub.

**Comment by insomniachi** [comment]
The code for this already exists, but i haven't enabled it because all the current providers have hard subbed videos, and it doesn't make sense to add external subtitles.

In the next release i'm adding zoro, it does have soft subs and i would be enabling this feature.
Also if you didn't know, you can stream torrents which will also be soft subbed.

TLDR, you'll need to use zoro/torrents for custom subtitles.

**Comment by Jakeverse** [comment]
Thank you so so much for releasing a new update so early.  I see you added lots of features, now Totoro is going to be world's best apps for anime on windows after Nekkoto in android. I hope this app will live and grow much bigger in the future and will gain lots of attention. ❤️
